src/main/java/mil/dds/anet/threads/AnetEmailWorker.java

```diff
@@ -131,19 +132,19 @@ protected void runInternal(Instant now, JobHistory jobHistory, Map<String, Objec
     emailContext.put("SUPPORT_EMAIL_ADDR", config.getDictionaryEntry("SUPPORT_EMAIL_ADDR"));
     emailContext.put("dateFormatter",
         DateTimeFormatter.ofPattern((String) config.getDictionaryEntry("dateFormats.email.date"))
-            .withZone(DaoUtils.getDefaultZoneId()));
+            .withZone(ZoneId.systemDefault()));
```

Comment:
the issue requests `It may be good to annotate times with the corresponding timezone.`. This can be done by changing the date string in `anet-dictionary.yml`. It would be good if this PR amends the example `anet-dictionary.yml` to do so.